.gitignore

```diff
@@ -94,3 +94,8 @@ tests/fsharpqa/Source/CodeGen/EmittedIL/QueryExpressionStepping/Utils.dll
 tests/fsharpqa/Source/CodeGen/EmittedIL/ComputationExpressions/ComputationExprLibrary.dll
 
 *.csproj.user
+
+*.ide
```

Comment:
VS 2015 generates temporary files for projects inside `vsintegration` folder. These files haven't been git-ignored yet.